README.md

```diff
@@ -6,6 +6,9 @@ This service is a lambda function which queries an external MongoDB instance for
 
 You'll need to download at least one of the existing datasets from the [GrantNav publisher page](http://grantnav.threesixtygiving.org/publisher/360G-blf) for the Fund.
 
+Make sure the CSV doesn't have a space as the very first character as this has been known to break imports in mysterious ways
```

Comment:
Any more context on this? Can we make the code more defensive to account for this?

Reply:
Hm, possibly? Could be a legacy of exporting from MS Excel – only way we could really fix this is test for the first character being a space. It's enough of an edge case that I'm happy to keep it a manual step for now, especially if the import process is likely to change in the near future.